python/delta/tests/test_sql.py

```diff
@@ -77,7 +77,7 @@ def test_generate(self):
         # check the contents of the manifest
         # NOTE: this is not a correctness test, we are testing correctness in the scala suite
         manifestPath = os.path.join(temp_file,
-                                    os.path.join("_symlink_format_manifest", "manifest"))
+                                    os.path.join("_symlink_format_manifest_presto", "manifest"))
```

Comment:
I think we should not change the existing API, this may break existing workloads, lets keep `symlink_format_manifest`